src/js/helpers/handleScroll.js

```diff
@@ -24,9 +24,10 @@ const setState = function (el, top, once) {
  * @param  {bool} once               plugin option
  * @return {void}
  */
-const handleScroll = function ($elements, once) {
-  const scrollTop = window.pageYOffset;
-  const windowHeight = window.innerHeight;
+const handleScroll = function ($elements, once, container) {
+  container = container || window;
+  var scrollTop = container == window ? window.pageYOffset : container.scrollTop;
```

Comment:
`scrollTop` and `windowHeight` are not redefined later. Use `const` instead. (note: duplicate of #393)